### ISSUE TITLE
Arm docker image

### ISSUE THREAD

**Issue by aberlaas** [damoeb/rss-proxy#37]
Hi,

rss-proxy is really promising !
Do you plan to make an Arm compatible docker image ?
If not, where do you think I should start to build such an image ? Build the container from code ?

Thanks

**Comment by damoeb** [comment]
Yeah, that sounds reasonable. I will check it out, give me one week

**Comment by aberlaas** [comment]
Ok, now I'm excited ! Thank you for helping revive RSS. I will try it as soon as it is available.
B.R.

**Comment by cesaryuan** [comment]
> Yeah, that sounds reasonable. I will check it out, give me one week

Hi, is there any updates?
I can't wait to use it on the ARM machine.🤣
Thanks

**Comment by cmakafui** [comment]
Hi,
Any updates on the Arm compatible docker image. 
Thanks

**Comment by jokaorgua** [comment]
Vote for ARM image!

**Comment by cesaryuan** [comment]
I try to make it compatible with arm64. Here is the QuickStart.

```bash
docker build -t damoeb/rich-rss:core-0 https://github.com/cesaryuan/rich-rss.git#master:packages/server-core/
wget https://raw.githubusercontent.com/cesaryuan/rss-proxy/master/chrome.json
wget https://raw.githubusercontent.com/cesaryuan/rss-proxy/master/docker-compose.yml
docker-compose up -d
```
In `docker-compose.yml` change `APP_PUBLIC_URL` accordingly which is the outfacing public url. 

Then open [localhost:8080](http://localhost:8080/) (`APP_PUBLIC_URL`) in the browser.

**Comment by qupb** [comment]
> ```shell
> docker-compose up -d
> ```

[+] Building 0.0s (0/0)                                                                                                         
unable to prepare context: path "./rss-proxy" not found

how to resolve this bugs？

**Comment by cesaryuan** [comment]
> how to resolve this bugs？

Sorry, forgot to update. Try delete `docker-compose.yml` and run `wget https://raw.githubusercontent.com/cesaryuan/rss-proxy/master/docker-compose.yml` again.

**Comment by BachDANG** [comment]
I tried making a rocker image on raspberry pi 4 but no success. Has anyone done that yet?
Is there a way to run it directly from the code without building docker ? 

Sorry for my limited understanding on this matter, but this is a great project.

**Comment by damoeb** [comment]
I pushed an arm image `damoeb/rss-proxy:2.1-arm`, can you give it a try?

**Comment by aberlaas** [comment]
Hi, I got the following error : `Unable to discover feeds: java.net.UnknownHostException: www.arte.tv` when I try to give it the following url : "https://www.arte.tv/fr/videos/plus-vues/"
The same error appears with any url I give. 

I ran it with the following command : 

`docker run -p 8080:8080 -e APP_API_GATEWAY_URL=http://[my_local_ip]:8080 -it damoeb/rss-proxy:2.1-arm`

**Comment by damoeb** [comment]
There is a bug when building docker images with buildx, that breaks dns resolution. Maybe they fixed it and I had an old version. Will check
